hash/msx1_cart.xml

```diff
@@ -674,14 +674,16 @@ Xyzolog - Electric Software
 		<year>1986</year>
 		<publisher>ASCII</publisher>
 		<info name="alt_title" value="日本語MSX-Write" />
-		<info name="usage" value="To display texts this needs to run on a machine with kanji support or have an external kanji cartridge mounted." />
 		<part name="cart" interface="msx_cart">
 			<feature name="pcb" value="MSX WRITE 900178B" />
 			<feature name="slot" value="msxwrite" />
 			<feature name="mapper" value="M60002-0125SP" />
 			<dataarea name="rom" size="524288">
 				<rom name="225 aa 8716 zoo" size="524288" crc="ef02e4f3" sha1="4180544158a57c99162269e33e4f2c77c9fce84e" offset="0" />
 			</dataarea>
+			<dataarea name="kanji" size="131072">
+				<rom name="kanjifont.rom" size="131072" crc="3b8fdf44" sha1="fc71561a64f73da0e0043d256f67fd18d7fc3a7f" />
+			</dataarea>
```

Comment:
Did this come with the Kanji cartridge?  Also, I’m a little concerned that putting this here will cause the Kanji cartridge to be automatically mounted on machines with two cartridge slots, even if they have an internal Kanji ROM.

Reply:
The kanji support is part of the MSX Write cartridge itself. The cartridge has a switch to disable the kanji part of MSX Write.

Reply:
More info about the MSX Write cartridge can be found here: https://www.msx.org/wiki/ASCII_Japanese_MSX-Write